src/selectors.cc

```diff
@@ -420,6 +420,13 @@ select_to_reverse(const Context& context, const Selection& selection,
                   Codepoint c, int count, bool inclusive)
 {
     auto& buffer = context.buffer();
+
+    // if we are selecting backwards from the beginning of the buffer,
+    // there is nothing more that can be selected.
+    if (selection.cursor() == buffer.begin()){
+        return {};
+    }
```

Comment:
the opening brace is still off (should be on a separate line), can you maybe remove the braces since it's just one statement